Source/rclUE/Public/ROS2Publisher.h

```diff
@@ -211,4 +211,7 @@ class RCLUE_API UROS2Publisher : public UActorComponent
 
     //! ROS2 publisher
     rcl_publisher_t RclPublisher;
+
+    //! Timer Manager
+    URRTimerManager* TimerManager;

```

Comment:
```suggestion
    UPROPERTY()
    URRTimerManager* TimerManager = nullptr;
```
This would save it from being Garbage collected, & thus also probably it's delegate being unbound I guess